app/src/main/java/com/example/cleanarchitecture/ui/viewmodels/OnBoardingViewModel.kt

```diff
@@ -18,14 +21,22 @@ class OnBoardingViewModel @Inject constructor(private val onBoardingUseCase: OnB
 
     init {
         viewModelScope.launch {
+//            Timber.i("datastore status", "${onBoardingUseCase.isOnBoardingFinished()}")
             _isOnBoardingFinished.value = onBoardingUseCase.isOnBoardingFinished()
+            Timber.d("OnboardingViewModel", "Onboarding finished status initialized: ${_isOnBoardingFinished.value}")
```

Comment:
${_isOnBoardingFinished.value}
always use sOnBoardingFinished.value while reading data
_isOnBoardingFinished is for updating data